test/integration/replicasets/test_setup.py

```diff
@@ -242,33 +242,12 @@ def test_setup_bootstrap_vshard(project_with_instances, cartridge_cmd):
 
     rpl_cfg_path = project.get_replicasets_cfg_path()
 
-    # create router replicaset
-    # vshard bootstrapping will fail
-    rpl_cfg = {
-        'router': {
-            'roles': ['vshard-router', 'app.roles.custom', 'failover-coordinator'],
-            'instances': [router.name],
-        }
-    }
-
-    write_conf(rpl_cfg_path, rpl_cfg)
-
-    cmd = [
-        cartridge_cmd, 'replicasets', 'setup',
-        '--bootstrap-vshard'
-    ]
-
-    rc, output = run_command_and_get_output(cmd, cwd=project.path)
-    assert rc == 1
-
-    assert "router... CREATED" in output
-    assert "Sharding config is empty" in output
-
-    assert_replicasets(rpl_cfg, admin_api_url)
-    assert not is_vshard_bootstrapped(admin_api_url)
-
```

Comment:
Why this block was removed?

Reply:
That test was removed because, bootstrap_vshard function with new version of cartridge won't return an error about missing config. Now, it will tell that function worked succesfully and user can check cluster to see if all bootstrapped. So, that test fail with new cartridge, it is also mentioned in an issue linked to Pull request.